### PR TITLE
Added a script for pulling a map of usernames and hashed emails

### PR DESCRIPTION
**Pull request by blarghmatey**
To be able to merge data between the open discussions application and the mailgun logs we need a mapping of user ids and hashed email addresses. This script generates a parquet file of that mapping and uploasds it to S3 for being picked up by AWS glue and exposed to AWS Athena.

### CODE REVIEW

**Comment by blarghmatey** [comment]
Once this is merged we will need to update the deploy for the MITx ETL scripts since they are now in a subdirectory of this repo.

File: mit-open/etl_settings.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+mit-open:
+  db_url: postgresql://<user>:<password>@<host>:5432/<dbname>
+  s3_bucket: mitodl-data-lake/mailgun
+  hash_salt: 
+  aws_access_key_id: 
+  aws_secret_access_key:
```
Comment (shaidar) [inline mit-open/etl_settings.yml]: Does this file not go into pillar similar to `mitx_etl`?

File: mit-open/email_mapping.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+from hashlib import sha256
+from sqlalchemy import create_engine
+import yaml
+# from csv import Writer
```
Comment (shaidar) [inline mit-open/email_mapping.py]: ?

File: mit-open/email_mapping.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+import s3fs
+
+with open('etl_settings.yml') as etl_settings:
```
Comment (shaidar) [inline mit-open/email_mapping.py]: Should there be a check to verify that the file exists?

File: mit-open/etl_settings.yml (hunk 1 above)
Comment (blarghmatey) [reply mit-open/etl_settings.yml]: It does, I was just putting this in as an example of the settings that are supported.

**Comment by shaidar** [comment]
👍

**Comment by blarghmatey** [comment]
I'm going to wait on merging this until we update the state for deploying the MITx ETL

**Pull request merged**